### ISSUE TITLE
startDate and endDate are formatted wrong for /Projects

### ISSUE THREAD

**Issue by jklarfeld** [projectjellyfish/api#518]
For the `/projects` end point, the dates for `startDate` and `endDate` are formatted incorrectly as `yyyy-mm-dd` when all other dates are formatted as ISO8601 compliant UTC.

**Comment by AllenBW** [comment]
ISO 8601, for representing JUST date (sans time component) is YYYY-MM-DD. 

http://en.wikipedia.org/wiki/ISO_8601

Now as for other date representations in jellyfish there is a time component added, for example 2015-04-19T12:59:23Z is the format you might see for when an order was created or updated at.

Presently we don't account for project start time in the creation of a project and given this is the real possibility of the beginning of a project being at a specific time in addition to a date the time component probably shouldn't be added on as the action is encountered by the JF DB (in a similar fashion to that of order placing)

TL;DR  @chriskacerguis, do we care about project start times.  if so I am signing myself up for work.  If not, there really isn't anything to change here, dates are already served as iso8601.

**Comment by chriskacerguis** [comment]
I'm not 100% sure that the 'time' will be important, but you never know.  So, let's say yes we care about the start time.

**Comment by mafernando** [comment]
@jklarfeld - based on the Allen's feedback, start/end_date should return dates not timestamps, is this an issue?

`start/end_date` should be returning ISO 8601 compliant dates - which it does, it just doesn't have an encoding of the time elapsed on that day like a traditional timestamp.

**Comment by jklarfeld** [comment]
Ideally, all date/time stamps would return in the same format, for consistency and also to ease parsing.

I can work with the sans-timestamp version, although parsing code will have to be rewritten and setup to detect the second date format.

That is of course provided that these formats are set in stone in their current form and will not change.

**Comment by chriskacerguis** [comment]
@mafernando could we add the time, just to be consistent?

**Comment by mafernando** [comment]
@jklarfeld - start/end_date should now come back as timestamps.

the necessary changes have been merged into master and deployed onto Heroku.
